bitway/chain.json

```diff
@@ -41,9 +41,9 @@
   },
   "codebase": {
     "git_repo": "https://github.com/bitwaylabs/bitway",
-    "recommended_version": "v2.0.0",
+    "recommended_version": "v2.0.1",
     "compatible_versions": [
-      "v2.0.0"
+      "v2.0.1"
```

Comment:
is v2.0.0 compatible with v2.0.1? I'm wondering if it should remain in the array...